lib/webidl2.js

```diff
@@ -671,13 +660,18 @@
         error("Unterminated stringifier");
     }
 
-    function identifiers(arr) {
+    function* identifiers() {
```

Comment:
Admittedly, I'm not a huge fan of using generators for finite lists where values are not consumed incrementally. If the consumer of the generator is always going to be an array spreading the iterator, then we might as well leave this as returning an array. 

However, I'm open to discussion on the rationale for why you'd like this integrate this change.

Reply:
I did this because generators can reduce a slight amount of boilerplate:

```js
function array() {
  const arr = [];
  for (;;) {
    arr.push(value);
  }
  return arr;
}

// vs 

function* generate() {
  for (;;) {
    yield value;
  }
}
```

But yeah, it's not a big deal here.

Reply:
If it's ok, I'd prefer just having the `const arr =[];`. It lowers the cognitive overhead a little bit.

Reply:
Done!